emspy/query/fltquery.py

```diff
@@ -493,47 +533,59 @@ def _boolean_filter(op, d):
 def _discrete_filter(op, d, flt):
 	fld_info = d[0]
 
-	if op in list(basic_ops.keys()):
-		# Single input basic coniditonal operation
-		t_op = basic_ops[op]
-		val_info = d[1]
-		vid  = flt.get_value_id(val_info['value'], field_id = fld_info['value'])
-		val_info['value'] = vid
-		fltr = _filter_fmt1(t_op, fld_info, val_info)
-
-	elif op in [" in ", " not in "]:
-		t_op = sp_ops[op]
-		val_list = [{'type':x['type'], 'value':flt.get_value_id(x['value'], field_id=fld_info['value'])}\
-					 for x in d[1:]]
-		inp = [fld_info] + val_list
-		fltr = _filter_fmt1(t_op, *inp)
+	if not isinstance(op, list):
+		if op in list(basic_ops.keys()):
+			# Single input basic coniditonal operation
+			t_op = basic_ops[op]
+			val_info = d[1]
+			vid  = flt.get_value_id(val_info['value'], field_id = fld_info['value'])
+			val_info['value'] = vid
+			fltr = _filter_fmt1(t_op, fld_info, val_info)
+
+		elif op.strip() in ["in", "not in"]:
+			t_op = sp_ops[op]
+			val_list = [
+				{'type': x['type'], 'value': flt.get_value_id(x['value'], field_id=fld_info['value'])}
+				for x in d[1:]
+			]
+			inp = [fld_info] + val_list
+			fltr = _filter_fmt1(t_op, *inp)
+		else:
+			raise ValueError("%s: Unsupported conditional operator for discrete field type." % op)
 	else:
-		raise ValueError("%s: Unsupported conditional operator for discrete field type." % op)
+		raise ValueError("%s: Unsupported conditional operators for discrete field type." % op)
 	return fltr
 
 
 def _number_filter(op, d):
+	if isinstance(op, string_types):
+		if op in basic_ops:
+			t_op = basic_ops[op]
+			fltr = _filter_fmt1(t_op, d[0], d[1])
+		else:
+			raise ValueError("%s: Unsupported conditional operator for number field type." % op)
+	elif isinstance(op, list):
+		if op[1] in between_ops[op[0]]:
+			t_op = between_ops[op[0]][op[1]]
+			fltr = _filter_fmt1(t_op, d[0], d[1], d[2])
+		else:
+			raise ValueError("%s: Unsupported between conditional operators for number field type." % op)
 
-	if op in basic_ops:
-		t_op = basic_ops[op]
-		fltr = _filter_fmt1(t_op, d[0], d[1])
-	else:
-		raise ValueError("%s: Unsupported conditional operator for number field type." % op)
 	return fltr
 
 
 def _string_filter(op, d):
-
-	if op in ["==", "!="]:
-		t_op = basic_ops[op]
-		fltr = _filter_fmt1(t_op, d[0], d[1])
-
-	elif op in [" in ", " not in "]:
-		t_op = sp_ops[op]
-		fltr = _filter_fmt1(t_op, *d)
-
+	if not isinstance(op, list):
+		if op in ["==", "!="]:
+			t_op = basic_ops[op]
+			fltr = _filter_fmt1(t_op, d[0], d[1])
+		elif op.strip() in ["in", "not in"]:
+			t_op = sp_ops[op]
+			fltr = _filter_fmt1(t_op, *d)
+		else:
+			raise ValueError("%s: Unsupported conditional operator for string field type." % op)
 	else:
-		raise ValueError("%s: Unsupported conditional operator for string field type." % op)
+		raise ValueError("%s: Unsupported conditional operators for string field type." % op)
```

Comment:
I'm kind of thinking the same as above for indentation/readability purposes.  Thoughts?

```python
if not isinstance(op, list):
    raise ValueError("")

if op in ["==", "!="]:
    t_op = basic_ops[op]
    fltr = _filter_fmt1(t_op, d[0], d[1])
elif op.strip() in ["in", "not in"]:
    t_op = sp_ops[op]
    fltr = _filter_fmt1(t_op, *d)
```